src/lib/ipfs.ts

```diff
@@ -9,6 +9,27 @@ interface FilestoreResponse {
   Hash: string
   Size: string
 }
+type EmptyObject = object
+type Peers = EmptyObject[]
+type PeersResponse = {
+  Peers?: Peers
+}
```

Comment:
saying Peers is an array of empty objects here but later there's `Peers: { Peer?: unknown }[]` , is it something like 
```
type PeersResponse = {
  Peers?: { Peer: object }[]
}
```

Reply:
I'm guessing, haven't looked at a real `peersResponse`

Reply:
it passed tests